WordPress/Classes/ViewRelated/Post/Prepublishing/PrepublishingSocialAccountsViewController.swift

```diff
@@ -48,14 +71,21 @@ class PrepublishingSocialAccountsViewController: UITableViewController {
         fatalError("init(coder:) has not been implemented")
     }
 
-    init(model: PrepublishingAutoSharingModel) {
+    init(blogID: Int,
+         model: PrepublishingAutoSharingModel,
+         coreDataStack: CoreDataStackSwift = ContextManager.shared,
+         blogService: BlogService? = nil) {
+        self.blogID = blogID
         self.connections = model.services.flatMap { service in
             service.connections.map {
-                .init(service: service.name, account: $0.account, keyringID: $0.keyringID, isOn: $0.enabled)
+                .init(service: service.name, account: $0.account, keyringID: $0.keyringID, isOn: false)
             }
         }
         self.shareMessage = model.message
-        self.sharingLimit = model.sharingLimit
+        self.sharingLimit = .init(remaining: 1, limit: 30)
```

Comment:
Looks like a testing change made it in here.
```suggestion
        self.sharingLimit = model.sharingLimit
```

Reply:
🤦🏼‍♂️ of course. Thanks!

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/21179/commits/d72d82fcb590fdbb6e180499110274487adc9212.